pkgs/development/compilers/julia/1.3.nix

```diff
@@ -114,13 +114,13 @@ stdenv.mkDerivation rec {
 
   LD_LIBRARY_PATH = makeLibraryPath [
     arpack fftw fftwSinglePrec gmp libgit2 mpfr blas openlibm
-    openspecfun pcre2
+    openspecfun pcre2 lapack
   ];
 
   enableParallelBuilding = true;
 
   doCheck = !stdenv.isDarwin;
-  checkTarget = "testall";
+  checkTarget = "";
```

Comment:
From your hydra log, it looks like this line just disables all tests. If that's what you want to do, you should change this to ``doCheck = false``, vs obfuscating this by setting ``checkTarget = ""``.
I'm not comfortable approving anything that disables ALL tests, especially for something as big as a programming language.

Reply:
thanks told me the reason. I will reset hard to HEAD~1. Only adding dependence in this PR.

Reply:
last build on [hydra test](http://221.4.35.244:8300/build/11/nixlog/1/tail)